src/services/mongoose/accounts.ts

```diff
@@ -62,8 +62,8 @@ export const AccountsRepository = (): IAccountsRepository => {
     try {
       const accounts: UserRecord[] = await User.find(
         {
-          title: { $exists: true, $ne: null },
-          coordinates: { $exists: true, $ne: null },
+          title: { $exists: true, $ne: undefined },
```

Comment:
did we do a migration forr this?

Reply:
no - migration for this... I guess we should check if this is an issue

Reply:
I have checked on bbw that the queries:
```
{ title: { $exists: true, $ne: null }, coordinates: { $exists: true, $ne: null} }
```
and 
```
{ title: { $exists: true, $ne: undefined }, coordinates: { $exists: true, $ne: undefined } }
```
yield identical results so I'd say we do not need a migration for this.